manifests/charts/base/templates/validatingwebhookconfiguration.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.base.enableValidation }}
```

Comment:
We already have an option: configValidation. It is just not used anywhere anymore